cypress/integration/e2e/homepage_spec.js

```diff
@@ -0,0 +1,94 @@
+describe('Home page tests', function()
+{
+    const newProjectName = 'Toms Team';
+
+    beforeEach(function() {
+        cy.visit('');
+        // Initialise Fixture objects
+        cy.fixture('elements.json').as('element');
+    })
+
+    it('Test to ensure the homepage displays as expected', function()
```

Comment:
When this test fails for a specific reason (one of the `should()`'s fails) does it tell you exactly which one failed? If not would it be worth structuring your tests like...
```
describe('Test to ensure the homepage displays as expected', () {
  it('should display appLogo', () => {
    cy.get(this.element.appHeaderElements.appLogo)
            .should('be.visible');
  });
});
```

If it is anything like Jasmine then I think you would prefer the way it looks when it fails.

I probably should have a look at how the results are displayed :) - Also, I imagine you have it just the way you like it haha

Reply:
Ideally yes you may want too, the problem is when viewing the cypress results it becomes too cluttered and provides too much info for passing builds.
But luckily when cypress (it's) fail they're displayed out in the cypress test run console

Reply:
So it seems like it displays it the way that Jasmin does - I would say you have a little too many checks in this one `it()` and the `it()` itself should become a `describe()`, but I wouldn't want to debate tests with a tester ;).

Reply:
Misunderstood, Sean explained and I will change it to make it easier to debug which one failed